README.md

```diff
@@ -26,17 +26,17 @@ Esta comunidade **não possui meios de conseguir renda**, porém, você pode doa
 
 ## O que é o Training Center
 
-O Training Center é uma comunidade de pessoas que trabalham na área de desenvolvimento de software (programação, design, bancos de dados etc.).
+O Training Center é uma comunidade de pessoas que trabalham na área de desenvolvimento de software: programação, design, bancos de dados, etc.
 
-Um de nossos principais objetivos é unir pessoas que querem entrar na área de desenvolvimento de software com gente que tem experiência para compartilhar e que podem ajudar essas pessoas a aprender e conseguir seu primeiro emprego. Também ajudamos pessoas que já trabalham na área e buscam aprimoramento profissional a elevar seus níveis técnicos ou soft skills para irem mais longe.
+Um de nossos principais objetivos é conectar pessoas que querem entrar na área com quem já atua. Compartilhando e ajudando as pessoas a aprenderem e conseguirem o seu primeiro emprego. Também ajudamos quem trabalha na área e busca o aprimoramento profissional para elevar seus níveis técnicos ou soft skills para irem mais longe.
```

Comment:
Acho que aqui `ajudando as pessoas a` estamos repetindo "pessoas" com o que está em "conectar pessoas", pode ser talvez `Compartilhando e as ajudando a aprenderem...`

Reply:
Podemos, só coloquei adicionei outro “pessoas” para reforçar, tendo está no início de outra frase